docs/pages-for-subheaders/monitoring-and-alerting.md

```diff
@@ -3,17 +3,18 @@ title: Monitoring and Alerting
 description: Prometheus lets you view metrics from your different Rancher and Kubernetes objects. Learn about the scope of monitoring and how to enable cluster monitoring
 ---
 
-Using the `rancher-monitoring` application, you can quickly deploy leading open-source monitoring and alerting solutions onto your cluster.
```

Comment:
The functional description should be one of the first things like it is currently or at least merged into the start of the sentence in the proposed change. Starting with the list of upstream projects isn't helpful for users unless they're already familiar with what each project does.